app/Product/Aspect/ExampleProductAspect.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Example\Project\Product\Aspect;
+
+use Go\Aop\Aspect;
```

Comment:
I love it! Especially how it'll be friggin impossible to debug!

"Why is it doing this?"
"Where is it coming from?"

Nobody knows since it's some obscure format of comment you're never gonna be able to find